heroic-elasticsearch-utils/src/main/java/com/spotify/heroic/elasticsearch/TransportClientSetup.java

```diff
@@ -36,21 +37,39 @@
     public static final String DEFAULT_CLUSTER_NAME = "elasticsearch";
     public static final List<String> DEFAULT_SEEDS = ImmutableList.of("localhost");
     public static final int DEFAULT_PORT = 9300;
+    public static final boolean DEFAULT_SNIFF = true;
```

Comment:
You're defaulting sniff to true here. I'm fine with having true as the default, but the example config lists it as false.

Reply:
I wanted to leave that as false. fixed it here.